tests/test_optim.py

```diff
@@ -1,119 +1,25 @@
 import pyhf
 import tensorflow as tf
+import pytest
 
 
-def test_optim_numpy():
+@pytest.fixture(scope='module')
+def source():
     source = {
-      "binning": [2,-0.5,1.5],
-      "bindata": {
-        "data":    [120.0, 180.0],
-        "bkg":     [100.0, 150.0],
-        "bkgsys_up":  [102, 190],
-        "bkgsys_dn":  [98, 100],
-        "sig":     [30.0, 95.0]
-      }
+        'binning': [2, -0.5, 1.5],
+        'bindata': {
+            'data': [120.0, 180.0],
+            'bkg': [100.0, 150.0],
+            'bkgsys_up': [102, 190],
+            'bkgsys_dn': [98, 100],
+            'sig': [30.0, 95.0]
+        }
```

Comment:
JSON uses double-quotes. Is there a reason to prefer single-quotes in python?

Reply:
No, that was my linter. I'll change it everywhere.

Reply:
hm, i would keep single quotes tbh. it's not a JSON string, it's a python dictionary (which might have been the result from a JSON parsing, but this is irrelevant I think), and the canonical quote for strings in python is single-quote

Reply:
> hm, i would keep single quotes tbh. it's not a JSON string, it's a python dictionary (which might have been the result from a JSON parsing, but this is irrelevant I think), and the canonical quote for strings in python is single-quote

Is there really a convention? Or is this just a code-style that we agree upon? If so, it's fine to leave it as single quotes. Just note that if it was in double-quotes, one could just copy/paste directly into a JSON without having to change anything (my 2 cents).